### ISSUE TITLE
EA 2226 and newer no longer work.

### ISSUE THREAD

**Issue by Koppajin** [pineappleEA/pineapple-src#41]
OS: Fedora 35
Kernel: Linux 5.14.18-301.fsync.fc35.x86_64 (copr sentry/kernel-fsync)
Mesa: 22.0.0-devel (copr gloriouseggroll/mesa-aco)

Upon trying to run 2226 and 2227 I get this in my terminal:

```
[jtfen@jfpc1 ~]$ /hdd/Emulation/Yuzu-EA-2226.AppImage
/usr/bin/sh: which: line 1: syntax error: unexpected end of file
/usr/bin/sh: error importing function definition for `which'
libGL error: MESA-LOADER: failed to open radeonsi: /usr/lib64/dri/radeonsi_dri.so: undefined symbol: amdgpu_query_video_caps_info (search paths /usr/lib64/dri, suffix _dri)
libGL error: failed to load driver: radeonsi
libGL error: MESA-LOADER: failed to open radeonsi: /usr/lib64/dri/radeonsi_dri.so: undefined symbol: amdgpu_query_video_caps_info (search paths /usr/lib64/dri, suffix _dri)
libGL error: failed to load driver: radeonsi
libGL error: MESA-LOADER: failed to open swrast: /usr/lib64/dri/swrast_dri.so: undefined symbol: amdgpu_query_video_caps_info (search paths /usr/lib64/dri, suffix _dri)
libGL error: failed to load driver: swrast
QPixmap::scaled: Pixmap is a null pixmap
QPixmap::scaled: Pixmap is a null pixmap
QPixmap::scaled: Pixmap is a null pixmap
libGL error: MESA-LOADER: failed to open radeonsi: /usr/lib64/dri/radeonsi_dri.so: undefined symbol: amdgpu_query_video_caps_info (search paths /usr/lib64/dri, suffix _dri)
libGL error: failed to load driver: radeonsi
libGL error: MESA-LOADER: failed to open radeonsi: /usr/lib64/dri/radeonsi_dri.so: undefined symbol: amdgpu_query_video_caps_info (search paths /usr/lib64/dri, suffix _dri)
libGL error: failed to load driver: radeonsi
libGL error: MESA-LOADER: failed to open swrast: /usr/lib64/dri/swrast_dri.so: undefined symbol: amdgpu_query_video_caps_info (search paths /usr/lib64/dri, suffix _dri)
libGL error: failed to load driver: swrast
QPixmap::scaled: Pixmap is a null pixmap
QPixmap::scaled: Pixmap is a null pixmap
yuzu: symbol lookup error: /usr/lib64/libvulkan_radeon.so: undefined symbol: amdgpu_query_video_caps_info
```
The last line comes from trying to open Emulation > Configure or trying to run a game and it immediately crashes. Is there some kind of crash dump I can share? I don't know where I'd find one.

**Comment by MGThePro** [comment]
Hmm, seems like #37 broke something. You can use [pineapple-linux](https://github.com/pineappleEA/pineapple-linux) in the meantime, that still works as it should.

**Comment by qurious-pixel** [comment]
It appears the AMD OpenCL drivers replace the libDRM driver, which does not have the symbols. `amdgpu_query_video_caps_info`

Seen here: [Redhat bug](https://bugzilla.redhat.com/show_bug.cgi?id=1964038) and here: [Lutris AMD driver bug](https://github.com/lutris/lutris/issues/3541) 

Will research this further.

**Comment by MGThePro** [comment]
Hmm, I don't think that's what's happening. I'm still getting the issue after removing every opencl related package.

**Comment by MGThePro** [comment]
should be fixed in 2238 with #42 . I'll make github actions rebuild the builds 2226-2237 just in case as well.
